src/main/java/com/stripe/model/Charge.java

```diff
@@ -606,6 +606,7 @@ public void setRuleObject(Rule rule) {
   @Setter
   @EqualsAndHashCode(callSuper = false)
   public static class TransferData extends StripeObject {
+    Long amount;
```

Comment:
Should we mark this with @Deprecated or only for payment-intent beta user only?
It will be removed in 8.0x with version pin.

Reply:
@mickjermsurawong-stripe No it won't be removed this is not limited by an API version for Charges and we will actively support it for a long time.

Reply:
or add comments to use `application_fee_amount` instead